app/views/storage/_required_storage.html.erb

```diff
@@ -0,0 +1,41 @@
+<%#-- copyright
+OpenProject is a project management system.
+Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See doc/COPYRIGHT.rdoc for more details.
+
+++#%>
+<% if storage.key?(:total) && storage[:total] > 0 %>
+<span class="tooltip--right"
+  data-tooltip="
+  <% (storage.keys - [:total]).each do |type| %>
+    <% if storage[type] > 0 %>
+    <%= l(type) %>: <%= number_to_human_size(storage[type], precision: 2) %>
+    &nbsp;
```

Comment:
If there only was a way to display native OpenProject tooltips: https://github.com/opf/openproject/pull/3246 :smile: 

Probably this will improve our situation once it is fixed... I am not sure of a proper solution either, sadly...